models/src_linkedin.yml

```diff
@@ -45,45 +45,33 @@ sources:
             description: Unique internal ID representing the creative.
 
           - name: last_modified_time
-            description: The timestamp of when the version was updated.
+            description: The timestamp of when the version was updated (Linkedin Ads pre-January 2023 API Update).
 
+          - name: last_modified_at
+            description: The timestamp of when the version was updated (Linkedin Ads post-January 2023 API Update).
+          
           - name: created_time
-            description: The timestamp of when the creative was created.
+            description: The timestamp of when the creative was created (Linkedin Ads pre-January 2023 API Update).
+
+          - name: created_at
+            description: The timestamp of when the creative was created (Linkedin Ads post-January 2023 API Update).
 
           - name: campaign_id
             description: The ID of the campaign the creative belongs to.
 
-          - name: status
+          - name: click_uri
```

Comment:
@fivetran-sheringuyen so that you are aware - the 'click_uri' field will contain data for both text and spotlight ad types. No other ad types contain data for this.

Reply:
@Csgoodman do you have an example of the structure of this updated field?

Reply:
circling back for documentation - it seems that a creative can only be of a single type -- dynamic ad, text or reference to InMail Content. So this field will follow essentially the same formatting and will only hold data for one of the previously mentioned ads (no need to worry about comma separated lists or anything!)